setup.cfg

```diff
@@ -0,0 +1,7 @@
+[flake8]
+ignore = B011, C814, E203, E402, E731, W503, W504, W391, C401, C408
+max-line-length = 99
```

Comment:
Don't we need this again for black to make them work together nicely? I thought black would warp to shorter lines, otherwise.

Reply:
good catch, I had a global `black` setting that hid the missing `pyproject.toml`. Added it now!